storage/namespace.go

```diff
@@ -225,6 +229,8 @@ func (n *dbNamespace) Bootstrap(
 	}
 	n.metrics.bootstrap.Success.Inc(1)
 
+	n.log.WithFields(namespaceLogField).Info("finished reading external data")
```

Comment:
It does provide timing breakdown of the different stages during bootstrap, will move it to bootstrap manager for now